examples/OrderEdit/OrderEditSetStagedActionsAction.json

```diff
@@ -0,0 +1,4 @@
+{
+  "action": "setStagedActions",
+  "stagedActions": "stagedOrderUpdateActionString"
```

Comment:
See comment at addStagedAction example. The structure should be a valid example.